gateway/src/main/java/com/lyft/data/gateway/ActiveClusterMonitor.java

```diff
@@ -98,8 +98,8 @@ private ClusterStats getPrestoClusterStats(ProxyBackendConfiguration backend) {
     try {
       URL url = new URL(target);
       HttpURLConnection conn = (HttpURLConnection) url.openConnection();
-      conn.setConnectTimeout((int) TimeUnit.SECONDS.toMillis(5));
-      conn.setReadTimeout((int) TimeUnit.SECONDS.toMillis(5));
+      conn.setConnectTimeout((int) TimeUnit.SECONDS.toMillis(15));
```

Comment:
Lets define an Enum for CONNECTION_TIMEOUT_MS and reference that here ?

Reply:
This is a one time change, we will refactor this class to decouple `monitor` from `StatsReporter` we can address it then if thats ok.